ride_share.rb

```diff
@@ -0,0 +1,73 @@
+# Create a ride share program:
+puts "Welcome to the ride share program!"
+
+# In this assignment, you should take data from a given CSV file and organize it into one single variable 
+# that is one single nested data structure. This nested data structure should use both arrays and hashes.
+total_rides = [
```

Comment:
This data structure worked well for you and you wrote efficient methods. Consider how the logic for your calculations would change if you were to organize your data by driver.

```ruby
total_rides = {
driver2_id: [{ride1_details}, {ride2_details},...]
driver1_id: [{ride1_details}, {ride2_details},...]
...
```
}